terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
timeout: the monitored command dumped core
./run_solvers.sh: line 122: 12693 Aborted                 timeout "$timeout_duration" "$solver_dir/open-wbo" "$cnf_file_old_format" > "$output_dir/${max_parallel_sessions}_session_open-wbo_output.txt"

real	28m54.278s
user	28m41.475s
sys	0m12.620s
